frontend/app/src/components/defi/Overview.vue

```diff
@@ -120,8 +129,18 @@ export default class Overview extends Vue {
   }
 
   get icon(): string {
+    // until we can check whether a file exists locally, let's make a whitelist of protocols where we prefer our own icon
+    const protocolWhitelist: string[] = ['aave', 'makerdao', 'compound'];
```

Comment:
Is it possible to add a todo here to change that to https://github.com/rotki/rotki/blob/c4db1aa42bc18b4086daca20b476881e94e46f81/frontend/app/src/services/defi/types.ts#L21 as soon as we add the Compound support?

Reply:
Sure